README.md

```diff
@@ -1 +1,39 @@
-# hello-microverse-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Pull Request should have a summary describing introduced changes in more detail, kindly add a description to your PR